i18n.py

```diff
@@ -15,14 +18,17 @@ def __init__(self, language=None):
             language = locale.getdefaultlocale()[
                 0
             ]  # getlocale can't identify the system's language ((None, None))
-        if not os.path.exists(f"./lib/i18n/{language}.json"):
+        base_dir = Path(__file__).parent.resolve()
+        language_file_path = os.path.join(base_dir, f"lib/i18n/{language}.json")
```

Comment:
Same, though if you're defining it here, just pass the path to `load_language_list`.

Reply:
Done